dcos/marathon.py

```diff
@@ -828,6 +828,36 @@ def test_for_pods(url, **kwargs):
         response = self._rpc.http_req(test_for_pods, 'v2/pods')
         return response.status_code // 100 == 2
 
+    def get_queued_app(self, app_id):
+        """Returns app information inside the launch queue.
+
+        :param app_id: the app id
+        :type app_id: str
+        :returns: app information inside the launch queue
+        :rtype: dict
+        """
+
+        response = self._rpc.http_req(http.get,
+                                      'v2/queue?embed=lastUnusedOffers')
+        app = next(
+            (app for app in response.json().get('queue')
+             if app_id == app.get('app', app.get('pod', {})).get('id')),
```

Comment:
i would keep it in this file, and just import it, since this is the "marathon common" file